src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -807,3 +875,60 @@ def _get_exon_number(tx_exons: List, tx_pos: int) -> int:
                 break
             i += 1
         return i
+
+    @staticmethod
+    def _get_adjacent_exon(
+        tx_exons_genomic_coords: List[Tuple[int, int, int, int, int]],
+        strand: Strand,
+        start: Optional[int] = None,
+        end: Optional[int] = None,
+    ) -> int:
+        """Return the adjacent exon given a non-exonic breakpoint. For the positive strand, adjacent is defined
+        as the exon preceeding the breakpoint for the 5' end and the exon following the breakpoint for the 3' end.
+        For the negative strand, adjacent is defined as the exon following the breakpoint for the 5' end and the exon
+        preceeding the breakpoint for the 3' end.
+
+        :param: tx_exons_genomic_coords: List of exons and genomic coords for a transcript
+        :param: start: Genomic coordinate of brekapoint
+        :param: end: Genomic coordinate of breakpoint
+        :return: Exon number corresponding to adjacent exon. Will be 1-based
+        """
+        if end and strand.value == strand.POSITIVE:
```

Comment:
You should be able to just do:

```suggestion
        if end and strand == strand.POSITIVE:
```